### ISSUE TITLE
Installing Lo/g/os

### ISSUE THREAD

**Issue by ghost** [install-logos/logos#33]
How exactly do you install Lo/g/os? I can boot from the live ISO, but I can't find any program or even any documentation on how to install the OS. Does Lo/g/os have an equivalent of Cnchi, Thus, or Ubiquity?

**Comment by Luminarys** [comment]
Logos development is on hold and right now we only have the liveboot ISO. Once RiceDB is done we'll go back to Logos, and an installer will be one of the features we implement. If you do want to install Logos, install Arch, all the listed packages, and copy over all the config files on the ISO.
